dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/DSpaceRestRepository.java

```diff
@@ -31,16 +40,28 @@
  * @author Andrea Bollini (andrea.bollini at 4science.it)
  *
  */
-public abstract class DSpaceRestRepository<T extends RestModel, ID extends Serializable>
+public abstract class DSpaceRestRepository<T extends RestAddressableModel, ID extends Serializable>
```

Comment:
As stated before, I'm not a fan of the "exception wrapping" that happens in this class and the use of directly using`RuntimeException`.

Reply:
As answer above, I prefer manage the exception in other PR.